<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="convertButton">
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="allDataRadioButton">
     <property name="text">
      <string>Only convert if all data is compatible</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QRadioButton" name="fixDataRadioButton">
     <property name="text">
      <string>Convert fixing data (Data not compatible will be changed to its default value.
Null data in a not null field will be changed to -9999)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="ConnectionWidget" name="widget_2" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="ConnectionWidget" name="widget" native="true"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select the conversion type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Conversion Log</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="5">
    <widget class="QPlainTextEdit" name="logDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="tabStopWidth">
      <number>200</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectionWidget</class>
   <extends>QWidget</extends>
   <header>DsgTools.CustomWidgets.connectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
